libraries/bezier-rs/src/subpath/solvers.rs

```diff
@@ -505,4 +618,153 @@ mod tests {
 	}
 
 	// TODO: add more intersection tests
+
+	#[test]
+	fn round_join_counter_clockwise_rotation() {
```

Comment:
Could you add a comment explaining the scenario this test case (and the one below) is covering? I noticed that the Subpath definitions are different and the offset distance is positive vs negative, but I don't really understand why and how that relates to being a clockwise / counter clockwise rotation